package.json

```diff
@@ -102,15 +102,15 @@
     "isbinaryfile": "5.0.0",
     "js-yaml": "4.1.0",
     "lodash": "4.17.21",
-    "mem-fs-editor": "9.5.0",
+    "mem-fs-editor": "github:mshima/mem-fs-editor#perf",
```

Comment:
Seems phishy. 🤔

Reply:
I will release a new mem-fs-editor with the fix.

Reply:
Released.